src/test/fuzz/net.cpp

```diff
@@ -0,0 +1,155 @@
+    while (fuzzed_data_provider.ConsumeBool()) {
+        switch (fuzzed_data_provider.ConsumeIntegralInRange<int>(0, 13)) {
+        case 0: {
+            node.CloseSocketDisconnect();
+            break;
+        }
+        case 1: {
+            node.MaybeSetAddrName(fuzzed_data_provider.ConsumeRandomLengthString(32));
+            break;
+        }
+        case 2: {
+            node.SetSendVersion(fuzzed_data_provider.ConsumeIntegral<int>());
+            break;
+        }
+        case 3: {
+            const std::vector<bool> asmap = ConsumeRandomLengthIntegralVector<bool>(fuzzed_data_provider, 128);
+            if (!SanityCheckASMap(asmap)) {
+                break;
+            }
+            CNodeStats stats;
+            node.copyStats(stats, asmap);
+            break;
+        }
+        case 4: {
+            node.SetRecvVersion(fuzzed_data_provider.ConsumeIntegral<int>());
+            break;
+        }
+        case 5: {
+            const CNode* add_ref_node = node.AddRef();
+            assert(add_ref_node == &node);
+            break;
+        }
+        case 6: {
+            if (node.GetRefCount() > 0) {
+                node.Release();
+            }
+            break;
+        }
+        case 7: {
+            if (node.m_addr_known == nullptr) {
+                break;
+            }
+            const std::optional<CAddress> addr_opt = ConsumeDeserializable<CAddress>(fuzzed_data_provider);
+            if (!addr_opt) {
+                break;
+            }
+            node.AddAddressKnown(*addr_opt);
+            break;
+        }
+        case 8: {
+            if (node.m_addr_known == nullptr) {
+                break;
+            }
+            const std::optional<CAddress> addr_opt = ConsumeDeserializable<CAddress>(fuzzed_data_provider);
+            if (!addr_opt) {
+                break;
+            }
+            FastRandomContext fast_random_context{ConsumeUInt256(fuzzed_data_provider)};
+            node.PushAddress(*addr_opt, fast_random_context);
+            break;
+        }
+        case 9: {
+            const std::optional<CInv> inv_opt = ConsumeDeserializable<CInv>(fuzzed_data_provider);
+            if (!inv_opt) {
+                break;
+            }
+            node.AddInventoryKnown(*inv_opt);
+            break;
+        }
+        case 10: {
+            const std::optional<CInv> inv_opt = ConsumeDeserializable<CInv>(fuzzed_data_provider);
+            if (!inv_opt) {
+                break;
+            }
+            node.PushInventory(*inv_opt);
```

Comment:
```
test/fuzz/net.cpp:115:18: error: no member named 'PushInventory' in 'CNode'

            node.PushInventory(*inv_opt);

            ~~~~ ^

test/fuzz/net.cpp:119:18: error: no member named 'PushBlockHash' in 'CNode'

            node.PushBlockHash(ConsumeUInt256(fuzzed_data_provider));

            ~~~~ ^

2 errors generated.

Reply:
Oh, thanks for letting me know! Now fixed!